spec/index.spec.js

```diff
@@ -14,59 +14,34 @@ describe("prismic-nuxt module", function() {
     }
 
     moduleOptions = {
-      endpoint: "http://test"
+      endpoint: "https://test.cdn.prismic.io/api/v2"
     };
   });
 
   it("should be defined", function() {
     expect(prismicNuxt).toBeDefined();
   });
 
-  it("should set __dangerouslyDisableSanitizersByTagID to an object", function() {
+  it("should add the prismic preview library", function() {
     prismicNuxt.call(context, moduleOptions);
-    expect(context.options.head.__dangerouslyDisableSanitizersByTagID).toEqual(jasmine.any(Object));
+    expect(context.options.head.script[0].src).toEqual('//static.cdn.prismic.io/prismic.min.js?repo=test&new=true');
+    expect(context.options.head.script[0].defer).toBeDefined();
+    expect(context.options.head.script[0].async).toBeDefined();
   });
 
-  it("should not set __dangerouslyDisableSanitizersByTagID to an object if already set", function() {
-    context.options.head.__dangerouslyDisableSanitizersByTagID = {};
-    prismicNuxt.call(context, moduleOptions);
-    expect(context.options.head.__dangerouslyDisableSanitizersByTagID).toEqual(jasmine.any(Object));
-  });
-
-  it("should disable sanitizer for prismic-nuxt", function() {
-    prismicNuxt.call(context, moduleOptions);
-    expect(context.options.head.__dangerouslyDisableSanitizersByTagID['prismic-nuxt']).toEqual(['innerHTML']);
-  });
-
-  it("should add the prismic endpoint script", function() {
-    prismicNuxt.call(context, moduleOptions);
-    expect(context.options.head.script[0]).toEqual({
-      hid: 'prismic-nuxt',
-      innerHTML: `window.prismic = {endpoint: '${moduleOptions.endpoint}'};`,
-      type: 'text/javascript',
-    });
+  it("should not add the prismic preview library if preview=false", function() {
+    prismicNuxt.call(context, { ...moduleOptions, preview: false });
+    expect(context.options.head.script).not.toBeDefined();
   });
 
-  it("should add the prismic library", function() {
+  it("should add the components", function() {
     prismicNuxt.call(context, moduleOptions);
-    expect(context.options.head.script[1].src).toEqual('//static.cdn.prismic.io/prismic.min.js');
+    expect(context.addPlugin.mock.calls).toHaveLength(3)
```

Comment:
This test is a little brittle because adding any more plugins would automatically break this test. Is there any way to check that the components have specifically been added? Maybe using `expect.stringContaining` or similar.